theme/less/gouvfr/site-search.less

```diff
@@ -1,5 +1,6 @@
 .site-search {
-    .input-group .form-control {
+    .input-group .form-control,
+    .input-group .form-control:last-child {
```

Comment:
Yes, it overrides default bootstrap style for input group (border and border-radius on right side).
This avoid style changes on component load